pom.xml

```diff
@@ -1127,12 +999,6 @@
                 <module>libraries-data-io-2</module>
             </modules>
 
-            <properties>
```

Comment:
Let's move these tags inside profiles, that is how we will restrict the Java version profile-wise.
This java.version should not be used anywhere inside in internal pom.xml except for the root pom.xml

Reply:
ok, reverted the tags changes.